doc/source/infra/ibm/step-zero-ibm.rst

```diff
@@ -34,12 +34,12 @@ Procedure:
      2. When configuring the new cluster, select the **Cluster type** and click **Create Cluster** to provision a Kubernetes cluster.
         2.1 In the case of a free cluster you will see something similar to:
 
-        .. image:: ../_static/images/ibm/create-free-kubernetes-cluster-ibm-cloud.png
+        .. image:: /_static/images/ibm/create-free-kubernetes-cluster-ibm-cloud.png
```

Comment:
@choldgraf, this will fail if /latest/en is added as a prefix path right? Would you recommend staying with relative references or use some other trick to reference relative to the base path which may be / or /latest/en, but when debugging locally typically is just / which make us think it works until we deploy to RTD?

Reply:
Is there a reason not to use a relative reference?

Reply:
yeah relative references should work fine here I believe

Reply:
Will revert them to relative refs, did not no of this issue. :)
Just wanted to make it future proof, so in case of change of structure the links still work.